cl/people_db/views.py

```diff
@@ -38,72 +39,78 @@ def view_person(request, pk, slug):
     positions = judicial_positions + other_positions
 
     # Use Solr to get relevant opinions that the person wrote
-    conn = ExtraSolrInterface(settings.SOLR_OPINION_URL, mode="r")
-    q = {
-        "q": f"author_id:{person.pk} OR panel_ids:{person.pk}",
-        "fl": [
-            "id",
-            "court_id",
-            "caseName",
-            "absolute_url",
-            "court",
-            "court_citation_string",
-            "dateFiled",
-            "docketNumber",
-            "citeCount",
-            "status",
-            "citation",
-        ],
-        "rows": 5,
-        "start": 0,
-        "sort": "dateFiled desc",
-        "caller": "view_person",
-    }
-    authored_opinions = conn.query().add_extra(**q).execute()
-    conn.conn.http_connection.close()
+    with Session() as session:
```

Comment:
Why not use the same session for all three of these?

Reply:
I figured it was best to do this refactoring initially while avoiding any logic changes regarding when sessions should close for the most part. Sharing the sessions would probably be a decent optimization.